cmd/draft/up.go

```diff
@@ -194,3 +212,29 @@ func defaultDraftEnvironment() string {
 	}
 	return env
 }
+
+func parseIgnores() (*ignore.Rules, error) {
+	// default ignored extensions
+	defaultsIR := strings.NewReader(strings.Join(defaultIgnores, "\n"))
```

Comment:
I would prefer if we wrote out a `.draftignore` with these default ignores in the file. That way it's less "black magic" and users can actually check for these files if they wanted to.